pyiode/objects/table.pyx

```diff
@@ -1232,12 +1253,18 @@ cdef class Table:
         index: int
             index where to insert a line.
         value: int or str or list(str) or tuple(str) or TableLine
+            # GDM> the \n in the following lines seem odd. Do they actually do anything in a docstring?
             value of the new line.\n
-            If int, 'value' represents the type of the new line: LINE_TYPE_FILES, LINE_TYPE_MODE, 
+            # GDM> I *wonder* if an explicit and optional line_type argument would not be cleaner for the 'int' use case.
+            If int, 'value' represents the type of the new line: LINE_TYPE_FILES, LINE_TYPE_MODE,
+            # GDM> LINE_TYPE_SEPARATOR instead of LINE_TYPE_LINE would be clearer IMO
```

Comment:
In the French documentation, the *separator* lines are specified as of type *LINE*.
Unfortunately, long-time IODE users are familiar with the term *LINE*.
So, I will keep *LINE* for the moment